hsctl

```diff
@@ -273,8 +273,8 @@ migrate_hs() {
 
 manage_py_hs() {
     echo "MANAGEPY:"
-    echo "  - docker exec -u hydro-service -ti hydroshare python manage.py ${@:2}"
```

Comment:
I'd prefer to leave -ti in... I use `./hsctl managepy shell` somewhat frequently.

Reply:
@sblack-usu, I absolutely want to keep it in `hsctl`. However, if we keep it, the script will not work on Windows in some cases.